letsencrypt/models/base_config_settings.py

```diff
@@ -0,0 +1,63 @@
+# -*- coding: utf-8 -*-
+# Copyright 2018 Therp BV <http://therp.nl>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from odoo import api, fields, models
+
+
+class BaseConfigSettings(models.TransientModel):
+    _inherit = 'base.config.settings'
+
+    letsencrypt_dns_provider = fields.Selection(
```

Comment:
it just occurs to me that it would make sense to allow configuring the extra domains on the wizard too. I'd suggest to read all `letsencrypt.altname.*` parameters in a text field, and on write split with whitespace into parameters again.

Reply:
...the one parameter per domain decision of mine is quite convoluted anyways. You could write the domains whitespace separated to a new parameter (`letsencrypt.altnames` or the like) and provide a migration script that collects the old ones